### DISCUSSION TITLE
Welcome to PokerGPT Discussions!

### DISCUSSION THREAD

**Discussion by HarperJonesGPT** [HarperJonesGPT/PokerGPT#1] [Announcements]
## 👋 Welcome!
Discuss anything related to PokerGPT here!
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NumseBacon** [comment]
yo, very cool project but for all newbies very hard to setup... Code doesnt work, and you yourself need to change paths and some other stuff, but i managed to fix all that, just to get hit with some api error, that i cant seem to get fixed, so if anyone figures it out lmk

**Reply by HarperJonesGPT** [reply]
Whats the API error?

Maybe you need to put your OpenAI api in there(if you havent)?

**Reply by NumseBacon** [reply]
Hello @HarperJonesGPT i managed to fix the error by changing some code. You think it would be possible to make the program automatically close the afk thing pop up?

**Reply by Pokerface989** [reply]
Hi! This is not for newbie. Many paths must be change. We have figured many things with my coder. I think we make this work when the program can read the cards. Any helps for that. We have the carbon thema as it looks like in picture, but can read like two cards only. Any help to regognize the cards, please? Tesseract should see them, we think, but what are the client setings that it can see them or what can be wrong. I really prechiate information? Could you @HarperJonesGPT help us a little bit, I would prechiate?

**Comment by Pokerface989** [comment]
Hi! This is not for newbie. Many paths must be change. We have figured many things with my coder. I think we make this work when the program can read the cards. Any helps for that. We have the carbon thema as it looks like in picture, but can read like two cards only. Any help to regognize the cards, please? Tesseract should see them, we think, but what are the client setings that it can see them or what can be wrong. I really prechiate information?

**Comment by Abemlol1092** [comment]
The program doesn't seem to work anymore. I have fixed all the flaws with the code. The program opens and rezises the poker client. Then it announces heroes position each round. It doesn't take any action automatically and seems to not be able to read other peoples action.
Please update or at least give us info that it doesn't work anymore
